ExampleMod/build.txt

```diff
@@ -1,8 +0,0 @@
-author = TML Team
-displayName = Example Mod
-homepage = https://tmodloader.net/
-hideCode = false
-hideResources = false
-includeSource = true
-buildIgnore = *.csproj, *.user, obj\*, bin\*, .vs\*, Old\*
-includePDB = true
```

Comment:
If a modder now gets ExampleMod and tries to build in-game, is it going to fail?

Reply:
It shouldn't fail